plugin/file/lookup.go

```diff
@@ -258,10 +270,8 @@ func (z *Zone) Lookup(ctx context.Context, state request.Request, qname string)
 			goto Out
 		}
 
-		ce, found := z.ClosestEncloser(qname)
-
 		// wildcard denial only for NXDOMAIN
-		if found {
+		if foundCe {
```

Comment:
is `found` actually still used?

Reply:
> is `found` actually still used?

Yes, in a few other places

https://github.com/coredns/coredns/blob/d6d8a9974bc45855a7d8a89e9e452cc9b6301d0e/plugin/file/lookup.go#L95

https://github.com/coredns/coredns/blob/d6d8a9974bc45855a7d8a89e9e452cc9b6301d0e/plugin/file/lookup.go#L162

https://github.com/coredns/coredns/blob/d6d8a9974bc45855a7d8a89e9e452cc9b6301d0e/plugin/file/lookup.go#L167

https://github.com/coredns/coredns/blob/d6d8a9974bc45855a7d8a89e9e452cc9b6301d0e/plugin/file/lookup.go#L250